cloud/pkg/cloudhub/handler/eventhandler.go

```diff
@@ -94,6 +95,15 @@ func (eh *EventHandle) HandleServer(container *mux.MessageContainer, writer mux.
 		return
 	}
 
+	// handle the reponse from edge
+	resourceSplits := strings.Split(container.Message.GetResource(), "/")
```

Comment:
Can  we use a regex here?
I think it would increase the readability.

Reply:
Thanks @subpathdev will update later:)